data_steward/utils/bq.py

```diff
@@ -286,3 +286,46 @@ def get_latest_validation_dataset_id(project_id):
             return sorted(validation_datasets, key=lambda x: x[0],
                           reverse=True)[0][1]
     return None
+
+
+def create_dataset(project_id,
+                   dataset_id,
+                   description=None,
+                   friendly_name=None,
+                   label_or_tag=None,
+                   overwrite_existing=None):
```

Comment:
I think `overwrite_existing=False` could work here. Is there any specific reason we need to use `None`?

Reply:
yep you're totally right, updating now!